rebar.config

```
@@ -11,7 +11,7 @@
 
 {deps, [
        gen_fsm_compat,
-       {riak_core, ".*", {git, "git://github.com/basho/riak_core.git", {branch, "develop-3.0"}}}
+       {riak_core, ".*", {git, "git://github.com/basho/riak_core.git", {branch, "mas-3.0-dialyzer1"}}}
```

If you merged riak_core PR, you can put this back in place

```suggestion
        {riak_core, ".*", {git, "git://github.com/basho/riak_core.git", {branch, "develop-3.0"}}}
```

---

Ta